files/image_config/monit/conf.d/sonic-host

```diff
@@ -20,3 +20,7 @@ check process rsyslog with pidfile /var/run/rsyslogd.pid
     start program = "/bin/systemctl start rsyslog.service"
     stop program = "/bin/systemctl stop rsyslog.service"
     if totalmem > 800 MB for 10 times within 20 cycles then restart
+
+check program routeCheck with path "/usr/bin/route_check.py"
+    every "*/5 * * * *"
```

Comment:
This *should* work as expected because we configure the Monit interval to 1 minute. However, according to the Monit [documentation](https://www.mmonit.com/monit/documentation/monit.html) the cron-style syntax is dependent upon the configured interval. For example, if the interval was set to the default (2 minutes), it could potentially miss running on minutes which are divisible by 5 ( Example: 00:02, 00:04, 00:06, 00:08, **00:10**). Here, this would not run for 10 minutes.

I suggest considering changing this to `every 5 cycles` to ensure it is run every 5 minutes.

Reply:
Thanks Joe! Updated as suggested.